client/commands/v2/pysa_server.py

```diff
@@ -129,60 +219,102 @@ async def process_did_save_request(
             raise json_rpc.InvalidRequestError(
                 f"Document URI is not a file: {parameters.text_document.uri}"
             )
+        await self.update_errors(document_path)
+
+    async def process_copy_model_request(
+        self, parameters: lsp.DidCopyModelParameters
+    ) -> str:
+        document_path = parameters.path
+        if document_path is None:
+            raise json_rpc.InvalidRequestError(
+                f"Document path is not a file: {parameters.path}"
+            )
+        return await self.copy_model(document_path, parameters.position)
 
     async def run(self) -> int:
         while True:
             async with _read_lsp_request(
                 self.input_channel, self.output_channel
             ) as request:
-
```

Comment:
This whole block of changes is actually just an indentation change plus the addition of the `"pysa/copyModel"` branch, right?

Reply:
Yep! Just my linter being overdramatic as usual 🙄